test.tsconfig.json

```diff
@@ -11,20 +11,11 @@
             "@cmt/*": ["src/*"],
             "@test/*": ["test/*"]
         },
-        "typeRoots": [
-            "node_modules/@types",
-            "types/"
-        ],
         "sourceMap": true,
-        "rootDir": ".",
-        "strict": true,
```

Comment:
why are these deleted here, but kept in `tsconfig.json`?

Reply:
typeRoots were the one that was unnecessary and I removed that one, I brought back the other ones so that the two config file stay similar.